src/ci/docker/host-x86_64/dist-ohos/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:23.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    g++ \
+    make \
+    ninja-build \
+    file \
+    curl \
+    ca-certificates \
+    python3 \
+    git \
+    cmake \
+    sudo \
+    gdb \
+    libssl-dev \
+    pkg-config \
+    xz-utils \
+    unzip \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY ohos-sdk-windows_linux-public.tar.gz /tmp 
```

Comment:
This line should probably be removed, as the file is downloaded on line 24? The file is not in the repo, so the `COPY` command will most likely fail.

Reply:
Oops, that was a leftover from testing. I'll set up a CI job to test this.